src/Sarif.UnitTests/TestData/PrereleaseCompatibilityTransformer/Inputs/WithExternalPropertyFiles.sarif

```diff
@@ -0,0 +1,56 @@
+﻿{
+    "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-10-10",
+    "version": "2.0.0-csd.2.beta.2018-10-10",
+    "runs": [{
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "results": [],
+      "externalPropertyFiles" : {
```

Comment:
>"externalPropertyFiles" : { [](start = 6, length = 27)

there is no test case which tests this code path so i generated a net new test, i have no idea if any of these values resemble a real world so please cross validate if this can be improved.